spot-contracts/contracts/vaults/RolloverVault.sol

```diff
@@ -283,46 +276,43 @@ contract RolloverVault is
     //--------------------------------------------------------------------------
     // External & Public read methods
 
-    /// @notice Fetches the vault's asset token balance.
     /// @param token The address of the asset ERC-20 token held by the vault.
-    /// @return The token balance.
-    function assetBalance(IERC20Upgradeable token) external view returns (uint256) {
+    /// @return The vault's asset token balance.
+    function vaultAssetBalance(IERC20Upgradeable token) external view returns (uint256) {
         return isVaultAsset(token) ? token.balanceOf(address(this)) : 0;
     }
 
-    /// @notice Checks if the given token is held by the vault.
-    /// @param token The address of a token to check.
-    function isVaultAsset(IERC20Upgradeable token) public view returns (bool) {
-        return token == underlying || _deployed.contains(address(token)) || address(perp) == address(token);
-    }
-
-    /// @notice Total count of deployed asset tokens held by the vault.
-    /// @return The count.
-    function deployedCount() public view returns (uint256) {
+    /// @return Total count of deployed asset tokens held by the vault.
+    function deployedCount() external view returns (uint256) {
         return _deployed.length();
     }
 
-    /// @notice The token address from the deployed asset token list by index.
     /// @param i The index of a token.
-    function deployedAt(uint256 i) public view returns (ITranche) {
-        return ITranche(_deployed.at(i));
+    /// @return The token address from the deployed asset token list by index.
+    function deployedAt(uint256 i) external view returns (IERC20Upgradeable) {
+        return IERC20Upgradeable(_deployed.at(i));
     }
 
-    /// @notice Total count of earned income tokens held by the vault.
-    /// @return The count.
-    function earnedCount() public pure returns (uint256) {
+    /// @return Total count of earned income tokens held by the vault.
+    function earnedCount() external pure returns (uint256) {
```

Comment:
Keeping these because you see them being part of the generalized vault interface?

Reply:
yea we'll need it when we add a secondary perp